mariadb/README.md

```diff
@@ -41,7 +41,8 @@ default database connection configuration.
 | :------------------- | :------ | :------------------------------------------------------------------------------------ |
 | MYSQL_ROOT_PASSWORD  |         | The database root user password. Defaults to `DB_ROOT_PASSWORD`                       |
 | MYSQL_ROOT_USER      |         | The database root user (used to create the site database). Defaults to `DB_ROOT_USER` |
-| MYSQL_MAX_ALLOWED_PACKET | 16777216 | Max packet length to send to or receive from the server, [documentation](https://mariadb.com/docs/server/ref/mdb/system-variables/max_allowed_packet/) |
+| MYSQL_MAX_ALLOWED_PACKET | 16777216 | Max packet length to send to or receive from the server, [documentation](https://mariadb.com/docs/server/ref/mdb/system-variables/max_allowed_packet/) 
+| TRANSACTION_ISOLATION | READ-COMMITTED | The isolation level for transactions.
```

Comment:
Same prefix note as above, and please link to the documentation.

https://mariadb.com/docs/server/ref/mdb/system-variables/transaction_isolation/